pexpect/pxssh.py

```diff
@@ -310,6 +317,13 @@ def login (self, server, username, password='', terminal_type='ansi',
             ssh_options = ssh_options + " -o'NoHostAuthenticationForLocalhost=yes'"
         if self.force_password:
             ssh_options = ssh_options + ' ' + self.SSH_OPTS
+        if ssh_config is not None:
+            try:
+                if spawn_local_ssh:
+                    os.path.isfile(ssh_config)
+            except:
```

Comment:
Catching all exceptions like this is not good practice, because it can obscure other problems. In this case it also won't do what you want, because `isfile()` returns False rather that raising an exception.

Maybe you want:

```python
if spawn_local_ssh and not os.path.isfile(ssh_config):
    raise ...
```

Reply:
Odd because I copied what was already there.
I'll push this into a function as well as I debating either to leave it as it is or functionize it.